Library/Homebrew/commands.rb

```diff
@@ -108,10 +108,15 @@ def internal_developer_commands_paths
   end
 
   def official_external_commands_paths(quiet:)
-    %w[bundle services test-bot].map do |cmd|
-      tap = Tap.fetch("Homebrew/#{cmd}")
+    OFFICIAL_CMD_TAPS.flat_map do |tap_name, cmds|
+      tap = Tap.fetch(tap_name)
       tap.install(quiet: quiet) unless tap.installed?
-      external_ruby_v2_cmd_path(cmd)
+      cmds.map do |cmd|
+        # TODO: ideally these would all be Ruby V2 commands (for improved
+        # command parsing and because `brew-command-not-found-init.rb` outputs
+        # when `require`d)
+        external_ruby_v2_cmd_path(cmd) || external_ruby_cmd_path(cmd) || external_cmd_path(cmd)
+      end
```

Comment:
```suggestion
        external_ruby_v2_cmd_path(cmd)
      end.compact
```

After https://github.com/Homebrew/homebrew-autoupdate/pull/46 and https://github.com/Homebrew/homebrew-command-not-found/pull/76 are merged, we won't need to have the fallbacks for this. I wouldn't be opposed to keeping them in just in case, though. It also may be smart to add the `.compact` so that we don't get any `nil`s (especially if we are removing the fallbacks)